app-admin/conky/conky-1.11.6_pre.recipe

```diff
@@ -0,0 +1,72 @@
+SUMMARY="A free, light-weight system monitor"
+DESCRIPTION="Conky is a free, light-weight system monitor for X, that displays \
+any kind of information on your desktop. \
+Conky can display more than 300 built-in objects, including support for: \
+
+• A plethora of OS stats (uname, uptime, CPU usage, mem usage, disk usage, \
+ \"top\" like process stats, and network monitoring,	just to name a few). \
+• Built-in IMAP and POP3 support. \
+• Built-in support for many popular music players (MPD, XMMS2, Audacious). \
+• Can be extended using built-in Lua support, or any of your own scripts \
+  and programs (more). \
+• Built-in Imlib2 and Cairo bindings for arbitrary drawing with Lua (more). \
+• Runs on Linux, FreeBSD, OpenBSD, DragonFlyBSD, NetBSD, Solaris, Haiku OS, \
+  and macOS! \
+and much much more. \
+Conky can display information either as text, or using simple progress bars and \
+graph widgets, with different fonts and colours."
+HOMEPAGE="https://github.com/brndnmtthws/conky"
+COPYRIGHT="original torsmo code: 2004, Hannu Saransaari and Lauri Hakkarainen
+All code since torsmo fork: 2005-2019 Brenden Matthews, Philip Kovacs, et. al."
+LICENSE="GNU GPL v3
+GNU LGPL v3
+MIT"
+REVISION="1"
+srcGitRev="26d23cb70718956493df7d3102dc3000f0f56685"
+SOURCE_URI="https://github.com/mmuman/conky/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="7c503bf23870475cc852ca247694a3dd6f43c56da13236025d52fd648d108ded"
+SOURCE_DIR="conky-$srcGitRev"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	conky = $portVersion
+	cmd:conky = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	lib:libcurl
+	lib:libintl
+	lib:liblua
+	lib:libncurses
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libcurl
+	devel:libintl
+	devel:liblua
+	devel:libncurses
+	"
+
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gawk
+	cmd:gcc
+	cmd:git
+	cmd:make
+	cmd:pkg_config
+	"
+
+BUILD()
+{
+	cmake -DCMAKE_INSTALL_PREFIX=$prefix -DDOC_PATH=$relativeDocDir \
+	-DMAN_PATH=$relativeManDir -DBUILD_X11=false .
```

Comment:
missing tab here.

Reply:
Any reason for using $relative* here?

Reply:
Without it, it doesn't build. See how it's structured in CMakeFiles.txt.